.github/workflows/ci.yml

```diff
@@ -55,10 +55,15 @@ jobs:
           -x test
 
     - name: Coverage
-      uses: codecov/codecov-action@v1
+      uses: Wandalen/wretry.action@v1.3.0
```

Comment:
Manually backported because the codecov version was incremented in a change that was not backported